docs/pre-alpha.md

```diff
@@ -0,0 +1,33 @@
+# *Pre-alpha* Peras
+
+## The trade-off in pre-alpha Peras
+
+*Pre-alpha* Peras is characterized by a trade-off between the following two desiderata:
+
+ - **Small latency until blocks can be boosted and hence benefit from faster settlement.**
+
+   Peras only enables improved settlement guarantees for a block if that block (or a descendant) is boosted due to a Peras certificate. One important Peras protocol parameter is the *block selection offset* `L`/`perasBlockMinSlots`. Honest nodes will only ever vote for blocks at the beginning of a Peras round that are least `perasBlockMinSlots` old. Therefore, for this purpose, a small value is desirable.
+
+   For example, if `perasBlockMinSlots = 300`, then a block only even has a chance to benefit from a boost after `300 slots = 5 minutes`, whereas for `perasBlockMinSlots = 60`, the block can already be boosted after a minute.
+
+ - **Resilience against weak attackers trying to force Peras into a cooldown.**
+
+   When a Peras round is unsuccessful, i.e. did not result in a certificate, Peras enters a lengthy cooldown period (likely >1 day) during which honest votes do not vote and therefore Peras is effectively disabled, returning to Praos temporarily.
```

Comment:
Would it be possible to explain briefly why the cooldown period is that long and isn't determined by the parameters mentioned above?

Reply:
Good point, I added a footnote referencing A.2.1 ("Attacks motivating cooldown periods and their structure") in the report :+1: